restapi/src/main/java/io/stargate/web/resources/RowResource.java

```diff
@@ -410,10 +416,8 @@ private static Object filterToValue(Object val, String column, Table tableData)
         .collect(Collectors.toList());
   }
 
-  private List<String> idFromPath(PathSegment id) {
-    MultivaluedMap<String, String> matrixParameters = id.getMatrixParameters();
-    List<String> values = new ArrayList<>(matrixParameters.keySet());
-    values.add(0, id.getPath());
-    return values;
+  private List<String> idFromPath(String path) {
+    String id = path.substring(path.lastIndexOf("/") + 1);
+    return Arrays.asList(id.split(";"));
```

Comment:
Since I had the same question about `/`, I think the answer is you need to URL-encode it.

I tested both with the branch:
```
$ curl -X GET 'http://127.0.0.2:8082/v1/keyspaces/test/tables/foo/rows/a%2Fb;1' -H 'Content-Type: application/json' -H"x-cassandra-token: $TOKEN"
{"count":1,"rows":[{"ks1":"a/b","ks2":1}]}

$ curl -X GET 'http://127.0.0.2:8082/v1/keyspaces/test/tables/foo/rows/a%3Bb;1' -H 'Content-Type: application/json' -H"x-cassandra-token: $TOKEN"
{"count":1,"rows":[{"ks1":"a;b","ks2":1}]}
```